### PR TITLE
Updated wordings around new Palworld mechanic

### PR DESCRIPTION
**Pull request by jammsen**

### CODE REVIEW

File: GAMES_LIST.md [hunk 1]
```diff
@@ -462,5 +462,4 @@ EOS does not provide players data.
 
 ### <a name="palworld"></a>Palworld
 Palworld support can be unstable, the devs mention the api is currently experimental.  
-To query palworld servers, the `RESTAPIEnabled` must be `True` in the configuration file, and you need to pass
-the `username` (currently always `admin`) and `password` (settings parameter) for it.
+To query Palworld servers, the `RESTAPIEnabled` setting must be `True` in the configuration file, and you need to pass the `username` (currently always `admin`) and the `adminpassword` (settings parameter) for it.
```
Comment (CosminPerRam) [inline GAMES_LIST.md:465]: So, `adminpassword` its what is in the game's configuration?
I think this needs another rewording to explicitly mention that you need to pass a `password` parameter to gamedig from the `adminpassword` setttings parameter.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@
 * Fix: BeamMP maxplayers that was displaying player count (By @dgibbs64 #551)
 * Fix: BeamMP filter servers by address, not host (By @Rephot #558)
 * Palworld - Replace old and broken protocol with the new one (#560)
+* Updated some wordings around the new Palworld mechanic
```
Comment (CosminPerRam) [inline CHANGELOG.md:19]: This isn't needed as we have already edited the palworld notes in the reimplementation PR.

Comment (jammsen) [reply CHANGELOG.md]: Removed it.

File: GAMES_LIST.md (hunk 1 above)
Comment (jammsen) [reply GAMES_LIST.md:465]: See https://github.com/jammsen/docker-palworld-dedicated-server/blob/develop/docs/ENV_VARS.md and https://tech.palworldgame.com/settings-and-operation/configuration - The Value is just called adminpassword or ArminPassword. People with administrative access to the gameserver should understand what is asked of them.

Comment (CosminPerRam) [reply GAMES_LIST.md:465]: I understand, but I meant that the wording isn't really great here, yes the value is called `adminpassword` but we need to pass it to gamedig as the `password` field.

**Pull request merged**